settlement/paypal/transform.go

```diff
@@ -0,0 +1,346 @@
+package paypal
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/brave-intl/bat-go/settlement"
+	"github.com/brave-intl/bat-go/utils/altcurrency"
+	uuid "github.com/satori/go.uuid"
+	"github.com/shopspring/decimal"
+)
+
+// CompleteSettlement marks the settlement file as complete
+func CompleteSettlement(args Args) error {
+	fmt.Println("RUNNING: complete")
+	if args.In == "" {
+		return errors.New("the 'in' flag must be set")
+	}
+	if args.Out == "./paypal-settlement" {
+		args.Out = "./paypal-settlement.json"
+		// return errors.New("the 'out' flag must be set")
+	}
+	payouts, err := ReadFiles(args.Out)
+	if err != nil {
+		return err
+	}
+	for i, payout := range *payouts {
+		payout.Status = "complete"
```

Comment:
I think we'll also want to set the provider id associated with the transaction ( aka hash )

Reply:
i believe provider id / hash can be the same. this is just to mark the transaction as finished. we currently have no way to tie from paypal txs back to eyeshade txs